### PR TITLE
Fix delete action in Google Blockly labs

### PR DESCRIPTION
**Pull request by fisher-alice**
Currently in our Google Blockly labs (like Poetry) if a user deletes a block, the next blocks are also removed. This is undesirable behavior and is different from what happens in CDO Blockly labs and in the Google Blockly playground. In the latter labs, if a user deletes a block, only the block and its child blocks (but not its next blocks) are removed.
In order to change this behavior, the `healStack` and `animate` parameters were added as expected arguments in the override of the dispose function. 

This was also impacting the cut action when using the [Google cross tab copy/cut/paste plugin](https://github.com/code-dot-org/code-dot-org/pull/48997) in that when a user 'cuts' a block, the next blocks were also deleted even though the next blocks were NOT included in the paste action.

Once this PR is merged, it will fix the behavior for the cut/paste actions with the plugin.

Screencast video before update:

https://user-images.githubusercontent.com/107423305/200917804-07b144e7-389f-4073-8efd-2a67971eea95.mp4

Screencast video after update:

https://user-images.githubusercontent.com/107423305/200917857-3d77e8f7-ada2-4be5-a209-b03625161e4a.mp4

## Links
[jira ticket - Fix delete action in Google Blockly labs](https://codedotorg.atlassian.net/browse/SL-388)

## Testing story
I tested locally on one of our Google Blockly labs. We are planning to add integration tests for functions that override Google Blockly functions, but we would like to merge this bug fix before Dance party migrates from CDO Blockly to Google Blockly. 

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by mikeharv — APPROVED** [review]

**Review by bencodeorg — APPROVED** [review]
LGTM -- I know you didn't add this override, but could you add a link to the function this is overriding in core blockly? or to some docs? Would be good to have some context as to what these parameters mean

**Comment by fisher-alice** [comment]
> LGTM -- I know you didn't add this override, but could you add a link to the function this is overriding in core blockly? or to some docs? Would be good to have some context as to what these parameters mean

Yes - will do!

File: apps/src/blockly/googleBlocklyWrapper.js [hunk 1]
```diff
@@ -272,9 +272,13 @@ function initializeBlocklyWrapper(blocklyInstance) {
     this.removeUnusedBlockFrame();
   };
 
+  // overridden dispose function definition at https://github.com/google/blockly/blob/develop/core/block_svg.ts#L863
```
Comment (bencodeorg) [inline apps/src/blockly/googleBlocklyWrapper.js:275]: Nit for next time -- copy a permalink (one of the options when you click on a line on github), so as these files change, we have a consistent reference to the logic you're referencing.

Comment (fisher-alice) [reply apps/src/blockly/googleBlocklyWrapper.js:275]: Oh interesting! Will definitely use a permalink next time - didn't know about this option. Thanks!

**Pull request merged**